[HDBScalarFunction] Cast error when synchronizing scalar function

```
[ERROR] [pool-242-thread-1] o.e.d.c.s.api.AbstractSynchronizer - Resource [/registry/public/test/test.hdbscalarfunction] skipped due to an error:
  class com.codbex.kronos.hdb.ds.model.hdbscalarfunction.DataStructureHDBScalarFunctionModel cannot be cast to class 
  com.codbex.kronos.hdb.ds.model.hdbtablefunction.DataStructureHDBTableFunctionModel 
  (com.codbex.kronos.hdb.ds.model.hdbscalarfunction.DataStructureHDBScalarFunctionModel and 
  com.codbex.kronos.hdb.ds.model.hdbtablefunction.DataStructureHDBTableFunctionModel are in unnamed module of loader 
  org.apache.catalina.loader.ParallelWebappClassLoader @7f0df8f0)
```

Probably the `hdbscalarfunction` model should extend the `hdbtablefunction` model. 

https://github.com/codbex/codbex-kronos/blob/main/modules/engines/engine-hdb/src/main/java/com/codbex/kronos/hdb/ds/model/hdbscalarfunction/DataStructureHDBScalarFunctionModel.java

--- a/modules/engines/engine-hdb/src/main/java/com/codbex/kronos/hdb/ds/model/hdbscalarfunction/DataStructureHDBScalarFunctionModel.java
+++ b/modules/engines/engine-hdb/src/main/java/com/codbex/kronos/hdb/ds/model/hdbscalarfunction/DataStructureHDBScalarFunctionModel.java
@@ -11,11 +11,11 @@
  */
 package com.codbex.kronos.hdb.ds.model.hdbscalarfunction;
 
-import com.codbex.kronos.hdb.ds.model.DataStructureModel;
+import com.codbex.kronos.hdb.ds.model.hdbtablefunction.DataStructureHDBTableFunctionModel;
 
 /**
  * The Class DataStructureHDBScalarFunctionModel.
  */
-public class DataStructureHDBScalarFunctionModel extends DataStructureModel {
+public class DataStructureHDBScalarFunctionModel extends DataStructureHDBTableFunctionModel {
 
 }

